pkg/validation-webhook/snapshot.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"reflect"
 
-	volumesnapshotv1beta1 "github.com/kubernetes-csi/external-snapshotter/client/v3/apis/volumesnapshot/v1beta1"
+	volumesnapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v3/apis/volumesnapshot/v1"
```

Comment:
The webhook needs to support both v1beta and v1

Reply:
Based on discussion today, we're keeping the storage version at v1beta, so the webhook needs to validate v1beta objects

Reply:
I'll test this.

Reply:
I've tested this.  You are right, this should still be using volumesnapshotv1beta1.  I modified the code now.  It works as expected.